### ISSUE TITLE
Hex-a-gone trials mode causes non-final Hex games to be counted as finals

### ISSUE THREAD

**Issue by knightry** [ShootMe/FallGuysStats#70]
The new game mode that released today treats all hexagone games as finals, even the pre-final rounds.

**Comment by culturemind** [comment]
also every win in pre finals counts as a crown win

**Comment by ShootMe** [comment]
Working on a fix, is just a display issue so don'\t worry about the numbers for now.

**Comment by MCHAWKING-cpu** [comment]
Can you make it so that the non-finals rounds of hexagone do not count towards the current finals hexagone qualify percentage as well? Or would that be not possible?

**Comment by ShootMe** [comment]
You will basically always get a final qual for this game mode. There is no way to tell if you are on the 'final' version that I can think of atm.

**Comment by culturemind** [comment]
I think im more concerned about win count, i have 23 wins in the last hour haha

**Comment by MCHAWKING-cpu** [comment]
> 
> 
> You will basically always get a final qual for this game mode (I will limit it to just one though). There is no way to tell if you are on the 'final' version that I can think of atm.

Ah that's too bad but no worries. Thanks for all the work you do.

**Comment by ShootMe** [comment]
v1.108 should have the win/final fix. I couldn't find a way to actually determine the real final so it will still always be one final qual for this game mode.

**Comment by culturemind** [comment]
thanks

**Comment by rainyillusionist** [comment]
Hmmm the game does differentiate the Survival Hex and the Final Hex though, the player.log isn't indicative of it?

**Comment by 123JLaney123** [comment]
Does the game not differentiate between the two? If it does not, can you not just set it so rounds 1/2 = Hex trials and round 3 = normal Hex final?

**Comment by ShootMe** [comment]
I found something in the log, it seems to have been added fairly recently that tells if this round is a final or not. Added the new logic to check for it, so your numbers should be correct once you update to 1.109

**Comment by 123JLaney123** [comment]
1.109 works perfectly for me, thanks. Is there any way to separate survival Hex from final Hex? (maybe name it (maybe name the round Hex Trials?) Because right now I'm using the practice profile instead of main profile so it doesn't ruin my Hex stats by making them higher than they should be.

**Comment by knightry** [comment]
+1 to @123JLaney123 suggestion. 1.109 is functionally correct but grouping non-final and final rounds of the same game together doesn't feel right for tracking purposes.

**Comment by Zephymastyx** [comment]
Updated from an older version to 1.109 (probably 1.100, last updated after Season 3 released) and it reset my wins to 0. Got a win just before updating that should have the final tag in the log, still a big fat zero displayed (with any filter).

**Comment by ShootMe** [comment]
When you update you only want to replace your exe if you are doing it manually. The data.db file contains all your stats and shouldnt be deleted, otherwise it will reset everything.

**Comment by Zephymastyx** [comment]
I did only replace the exe. All my stats are still there, only the wins are not counted as such.

**Comment by ShootMe** [comment]
Youll have to upload your data.db file as I haven't encountered any issues with wins from anyone else that I know of, and my stats also are still fine.

**Comment by Zephymastyx** [comment]
Uploaded. Interestingly, it also displays "Finals 0/(total shows)", but a final streak of 1. My best final streak also changed to 4, which is probably my best final streak since hexagone trials. Was 13 before hexagone trails (and an unrealistic high number after hexagone trials, before updating)

[data.zip](https://github.com/ShootMe/FallGuysStats/files/5784360/data.zip)

**Comment by ShootMe** [comment]
Oh, you haven't updated in a very long time lol. It didn't recognize the version yuor data.db was on and wasn't able to upgrade it properly. This is the updated version. Should show correctly now for you.
[data.zip](https://github.com/ShootMe/FallGuysStats/files/5784415/data.zip)

**Comment by Zephymastyx** [comment]
Looks good, thanks!

**Comment by 123JLaney123** [comment]
@ShootMe Any response to my last suggestion within this post?
(sorry to be a bother).

**Comment by ShootMe** [comment]
Yea, I'm not quite sure how I feel about it. I will probably mess around with a few things this weekend and maybe add that in.

**Comment by Zephymastyx** [comment]
I agree with splitting Hexagone final and survival would be helpful, the qualification / win rates are vastly different (for most players at least) and having those mixed gives inaccurate results.
Also keep in mind there will probably be more variations of finals / non finals with Slime Climb trials being a possibility mentioned in the sub, so it will probably be a recurring issue.

**Comment by 123JLaney123** [comment]
That's my point. They're essentially different rounds.
